addons/website_event/views/event_templates_list.xml

```diff
@@ -147,11 +147,23 @@
     <t t-if="opt_events_list_columns" t-set="opt_event_size" t-value="opt_index_sidebar and 'col-md-6' or 'col-md-6 col-lg-4'"/>
     <t t-else="" t-set="opt_event_size" t-value="opt_index_sidebar and 'col-12' or 'col-xl-10 offset-xl-1'"/>
     <!-- No events -->
-    <t t-if="not event_ids">
+    <t t-if="not event_ids"> 
         <div class="col-12">
-            <div class="h2 mb-3">No events found.</div>
-            <div class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Use the top button '<b>+ New</b>' to create an event.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="mb-3">
+                    <t t-call="website_event.event_svg"/>
+                </div>
+                <h3 class="mb-2">No events scheduled yet</h3>
+                <p t-if="search or searches and searches['search']">We couldn't find any event matching your <strong t-out="search or searches and searches['search']"/> search.</p>
+                <p t-else="">We couldn't find any event scheduled at this moment.</p>
+            </div> 
+            <div class="o_not_editable d-flex flex-column align-items-center justify-content-between my-3" groups="event.group_event_user">
+                <div class="d-flex flex-column align-items-center justify-content-between">
+                    <a class="o_wevent_cta btn"> <!-- Add working link to event modal creation -->
```

Comment:
With your suggestion, I'm going straight to the App drawer/switcher instead of the new event form view

Reply:
Hm, I could have sworn it worked. I'll try to figure out how to create a link for that then. I don't think you can just create a link to the name of the view (because it doesn't define an action).

Ping me if you don't hear anything by the end of the week :)

Reply:
I couldn't find a nice href solution. So I think we'll need to pass a new value to the render, no biggie.

Here's a commit that works:
a594eaa2e2b0f2fbd2b1c4c7d00c8322afffa2f5